lms/djangoapps/ccx/views.py

```diff
@@ -93,6 +93,12 @@ def wrapper(request, course_id):
 
         if not course.enable_ccx:
             raise Http404
+        elif configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and not run_extension_point(
+            'PCO_IS_INSTITUTION_ADMIN_AND_LICENSE',
+            user=request.user,
+            master_course=course.id,
+        ):
+            raise Http404
```

Comment:
Don't we have to validate if we are working with a Master Course here?. @anfbermudezme

Reply:
It is not necessary since the validation must be done for both, CCXs and master courses.